### DISCUSSION TITLE
See Through Exit End Portal

### DISCUSSION THREAD

**Discussion by TheBrokenRail** [iPortalTeam/ImmersivePortalsMod#1599] [Ideas]
Also don't give the levitation effect to players in creative mode.

**Comment by qouteall** [comment]
Currently, the architecture of this mod does not support non-square portal. To support this I need to rewrite a lot. Not planned.

**Comment by TheBrokenRail** [comment]
Is this possible now?

**Comment by qouteall** [comment]
yes

**Comment by ImSpaceLover** [comment]
Is it possible now with the /portal multidest?

**Comment by LoganDark** [comment]
> Is it possible now with the /portal multidest?

That has nothing to do with it. Multidest only manages which players can go through the portal, not how the portal itself is shaped. You can already manually build a portal using Portal Helpers to match the shape of the end portal.

**Comment by ImSpaceLover** [comment]
I mean, Exit End Portal now can teleport every player to his own spawnpoint and not to 0 64 0

**Comment by LoganDark** [comment]
Could be good for a custom portal subclass
